app/javascript/mastodon/reducers/push_notifications.js

```diff
@@ -1,17 +1,17 @@
 import { STORE_HYDRATE } from '../actions/store';
-import { SET_BROWSER_SUPPORT, SET_SUBSCRIPTION, CLEAR_SUBSCRIPTION, ALERTS_CHANGE } from '../actions/push_notifications';
+import { SET_SUBSCRIPTION, CLEAR_SUBSCRIPTION, ALERTS_CHANGE } from '../actions/push_notifications';
 import Immutable from 'immutable';
+import { supportsDesktopNotifications } from '../agent';
 
 const initialState = Immutable.Map({
   subscription: null,
   alerts: new Immutable.Map({
-    follow: false,
-    favourite: false,
-    reblog: false,
-    mention: false,
+    follow: !supportsDesktopNotifications(),
```

Comment:
Can't check right now, but I think that Firefox for Android supports regular notifications, so this will be disabled.

Reply:
I heard that they are enabled for mobile devices because they do not support regular notifications, so I thought they should be disabled in the case. Is that wrong?

Reply:
I could not find any comments saying so. Probably that's just my misconception. I'll fix it soon.